language/compiler/bytecode-source-map/src/source_map.rs

```diff
@@ -549,6 +567,13 @@ impl<Location: Clone + Eq> SourceMap<Location> {
                 .dummy_struct_map(&module, &struct_def, default_loc.clone())?;
         }
 
+        for const_idx in 0..module.constant_pool().len() {
+            empty_source_map.add_const_mapping(
+                ConstantPoolIndex(const_idx as TableIndex),
+                ConstantName::new(format!("CONST{}", const_idx)),
+            )?;
+        }
```

Comment:
@tzakian thoughts? I'm not familiar with this area

Reply:
This makes sense to me. Since there isn't a source map there isn't really a way of getting the name of the constant. This is similar to what is done for type arguments/function arguments if a source map doesn't exist.